setup-data-tokeninzation-solution-v2.sh

```diff
@@ -57,6 +59,11 @@ gcloud projects add-iam-policy-binding ${PROJECT_ID} --member="serviceAccount:${
 
 # trigger the cloud build script to create DLP templates
 gcloud builds submit . --config dlp-demo-part-2-dlp-template.yaml --substitutions _KEK_CONFIG_FILE=gs://${DATA_STORAGE_BUCKET}/${KEK_FILE_NAME},_GCS_BUCKET_NAME=gs://${DATA_STORAGE_BUCKET},_API_KEY=$(gcloud auth print-access-token)
+
+# create Pub/Sub topic and enable GCS notification to the pub/sub topic
+gcloud pubsub topics create ${GCS_NOTIFICATION_TOPIC}
```

Comment:
Will the topic get deleted after the pipeline is stopped? If not, will re-running the script give error for duplicate topic creation?

Reply:
The setup script has been designed to run only once on a fresh project and reuse the resources. Hence the topic won't be deleted after job completion